mod/wikirate_project/set/self/about_us.rb

```diff
@@ -7,4 +7,12 @@
   def breadcrumb_title
     "About us"
   end
+
+  def social_image
+    Card[:image_on_socials]&.format(:text)&.render_source
+  end
+
+  def social_description
+    Card.fetch(:social_description)&.format(:text)&.text_description
+  end
 end
```

Comment:
For the repeated code (the `social_image`), the preferred method is to make an abstract set.  Eg abstract/social_image.rb.  Then you use `include_set Abstract::SocialImage` in these files.

For the descriptions, is there a reason not to use the default behavior and create an `:about_us+:description` card? If that's already taken, you could generalize this to use a different field in the abstract mod I just mentioned.

Reply:
I have addressed both your comments on the following commits.